10_deploy_rook.sh

```diff
@@ -7,27 +7,43 @@ source common.sh
 figlet "Deploying rook" | lolcat
 eval "$(go env)"
 
+CEPH_VERSION=v14
+export MIXINPATH="$GOPATH/src/github.com/ceph/ceph-mixins"
 export ROOKPATH="$GOPATH/src/github.com/rook/rook"
 cd $ROOKPATH/cluster/examples/kubernetes/ceph
-oc create -f common.yaml
-sed '/FLEXVOLUME_DIR_PATH/!b;n;c\          value: "\/etc/kubernetes\/kubelet-plugins\/volume\/exec"' operator-openshift.yaml > operator-openshift-modified.yaml
-sed -i 's/# - name: FLEXVOLUME_DIR_PATH/- name: FLEXVOLUME_DIR_PATH/' operator-openshift-modified.yaml
+
```

Comment:
we still need oc create -f common.yaml ;)

Reply:
There's common-modified.yaml around line 17.